Editor/GameLiftClientSettingsEditor.cs

```diff
@@ -49,22 +45,46 @@ public override void OnInspectorGUI()
             }
         }
 
-        private void EditLocalMode(GameLiftClientSettings targetSettings)
+        private void OnEnable()
+        {
+            _remoteUrl = serializedObject.FindProperty(nameof(GameLiftClientSettings.ApiGatewayUrl));
+            _poolClientId = serializedObject.FindProperty(nameof(GameLiftClientSettings.UserPoolClientId));
+            _region = serializedObject.FindProperty(nameof(GameLiftClientSettings.AwsRegion));
+            _isAnywhereTest = serializedObject.FindProperty(nameof(GameLiftClientSettings.IsAnywhereTest));
+            _computeName = serializedObject.FindProperty(nameof(GameLiftClientSettings.ComputeName));
+            _fleetID = serializedObject.FindProperty(nameof(GameLiftClientSettings.FleetID));
+            _fleetLocation = serializedObject.FindProperty(nameof(GameLiftClientSettings.FleetLocation));
+            _profileName = serializedObject.FindProperty(nameof(GameLiftClientSettings.ProfileName));
+        }
+
+        private void EditAnywhereMode(GameLiftClientSettings targetSettings)
         {
-            EditorGUILayout.LabelField("GameLift Local URL", targetSettings.LocalUrl);
-            EditorGUILayout.PropertyField(_localPort, new GUIContent("GameLift Local Port", "This port should match the port value defined in Local Testing"));
+            EditorGUILayout.PropertyField(_computeName, new GUIContent("Compute Name", "Fleet Compute Name"));
+            EditorGUILayout.PropertyField(_fleetID, new GUIContent("FleetID", "This Fleet Id should match the value generated in the amazon console"));
+            EditorGUILayout.PropertyField(_fleetLocation, new GUIContent("Fleet Location", "This Location should match the value defined in your amazon console"));
+            EditorGUILayout.PropertyField(_profileName, new GUIContent("Profile Name", "This Name should match the value defined by your amazon account"));
```

Comment:
`generated in the amazon console` - at what point is this generated in the console from a customer perspective? We should consider alternative wording to make customers know where to procure this information from